web3/service.js

```diff
@@ -0,0 +1,102 @@
+import Web3 from 'web3';
+import { abi } from './abi';
+
+const TEST_PROVIDER_URL = 'https://ropsten.infura.io/v3/da3111c149bb4134a9b3439392abe518';
```

Comment:
This shouldn't really be committed, it's our private INFURA API key. Both this and `ADDRESS` could be in an `.env` file or parametrized in some other way good for React Native apps.

Reply:
ok